src/pretix/static/pretixbase/js/asynctask.js

```diff
@@ -218,10 +218,16 @@ $(function () {
             'this page and try again.'
         ));
 
-
-        console.log($(this).get(0))
         var formData = new FormData($(this).get(0))
         formData.append('ajax', '1');
+        if (e.originalEvent.submitter) {
+            // Not supported on IE, may lead to wrong results, but we don't support IE in the backend anymore
+            console.warn('Unknown submitter, IE unsupported');
+            var $submitter = $(e.originalEvent.submitter);
+            if ($submitter.attr("name") && $submitter.val()) {
+                formData.append($submitter.attr("name"), $submitter.val());
+            }
+        }
         $.ajax(
             {
                 'type': 'POST',
```

Comment:
```suggestion
                'type': this.method,
```

Reply:
From JQuery documentation, I'm unclear if there *might* be an issue with lower-case "post" instead of "POST" which is probably technical possibly with this.method, so I'd skip touching it until we need to

Reply:
Okay, might probably come from the original XMLHttpRequest, which also states methods in uppercase. Then we leave it as is and change later, if needed. Or we use `this.method.toUpperCase()`?